### ISSUE TITLE
how to resume trainning for unpaired train?

### ISSUE THREAD

**Issue by Aike2071** [GaParmar/img2img-turbo#132]
I tried replacing --pretrained_model_name_or_path with my checkpoints path but it seems that it's not working.The loss curves are just the same as that trained from stabilityai/sd-turbo.
